README.md

```diff
@@ -12,37 +12,45 @@ MODEL2CODE has been tested on Ubuntu 22.04 with Python 3.10.12.
 
 MODEL2CODE requires the following dependencies:
 
-* [TinyXML2](https://github.com/leethomason/tinyxml2/) (Tested with v10.0.0)
+* [TinyXML2](https://github.com/leethomason/tinyxml2/) (Tested with v10.0.0) for XML file handling.
 
 To install see [TinyXML2 Installation](https://github.com/leethomason/tinyxml2/tree/master?tab=readme-ov-file#building-tinyxml-2---using-vcpkg)
 
 ## Compiling
-To compile the code run the following commands
+To compile the code run the following commands:
 ```
 git clone https://github.com/convince-project/model2code.git
-cd model2code/code
+cd model2code
 mkdir build
 cd build
 cmake ../code
```

Comment:
Here maybe is better a general CMakeLists file in the root directory? just because it is normally the standard, and also I would rename the folder to src instead of code